### ISSUE TITLE
Expected release date?

### ISSUE THREAD

**Issue by RocketMike98** [NVlabs/FoundationPose#3]
Amazing work!! Any aprox code release date?

**Comment by wenbowen123** [comment]
Hi, our target release date is in March.

**Comment by Hitlzw** [comment]
Will the code be released as scheduled in March?

**Comment by wenbowen123** [comment]
current target: after mid March.

**Comment by ysdagar** [comment]
Hi Amazaing work! Can't wait to get my hands on this :)

Just checking, if the march release plan still holds good or if there is any delay?

**Comment by graboosky** [comment]
Looks amazing!!! Is it for real-time application?

**Comment by wenbowen123** [comment]
@graboosky yes, the original pytorch version runs 20-30Hz. An accelerated version will also be released afterwards.

**Comment by Jianxff** [comment]
Hi, great work! Will the code be released as scheduled now?

**Comment by wenbowen123** [comment]
Our code has released
